lib/knapsack_pro/base_allocator_builder.rb

```diff
@@ -101,5 +87,24 @@ def get_slow_test_files
       KnapsackPro.logger.debug("Detected #{slow_test_files.size} slow test files: #{slow_test_files.inspect}")
       slow_test_files
     end
+
+    def rspec_test_file_example_paths_for_slow_test_files(slow_test_files)
```

Comment:
If you move this into the adapter, I guess you could do:

```suggestion
    def example_paths_for(test_files)
```

Reply:
done.